src/Storage/Storage.Management/ChangeLog.md

```diff
@@ -18,6 +18,8 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
+* Updated parameter descriptions for Set-AzStorageLocalUser
+    - `Set-AzStorageLocalUser`
```

Comment:
No change log is needed for doc changes.

Reply:
Fixed